### DISCUSSION TITLE
ISO version

### DISCUSSION THREAD

**Discussion by Obese-obama** [thatskymod/Sky-CotL-Scripts#40] [Ideas]
Is it possible to make a iso version of this file instead of an APK for iOS users.

**Comment by Obese-obama** [comment]
Or .APA

**Comment by thatskymod** [comment]
Neither **ISO** *(optical disc image)* nor **APA** *(RSView Development Project Archive)* files are typically compatible with standard Apple devices.

For jailbroken iPhones, the most commonly used file formats include IPA (iOS App Store Package) for application installation, DEB files for Debian packages (often utilized for tweaks and software not available on the App Store), and various iOS-specific formats such as PLIST for configuration settings.
